streams-tests/jvm/src/test/scala/zio/stream/ZSinkPlatformSpecificSpec.scala

```diff
@@ -22,6 +23,18 @@ object ZSinkPlatformSpecificSpec extends ZIOBaseSpec {
           }
 
       }
+    ),
+    suite("fromOutputStream")(
+      test("writes to byte array ouput stream") {
+        val data = (0 to 100).mkString
+
+        for {
+          bytes  <- Task(data.getBytes("UTF-8"))
+          os      = new ByteArrayOutputStream(data.length)
+          length <- ZStream.fromIterable(bytes).run(ZSink.fromOutputStream(os))
+          str    <- Task(os.toString())
```

Comment:
To avoid system-dependent issues, let's explicitly set `UTF-8` charset for the `toString` here because this way it uses the default charset which may not be the same as the one you passed for `getBytes`